### DISCUSSION TITLE
allow 'enabled' to be available from env

### DISCUSSION THREAD

**Discussion by nabeelsaabna** [renovatebot/renovate#23413] [Suggest an Idea]
### Tell us more.

Make the `enabled` config option available from `env`

https://github.com/renovatebot/renovate/blob/6b117465590c0acb51d6ef65afd57325ae981d78/lib/config/options/index.ts#L221-L228

last updated on 2017 😮 

I do understand it is less intuitive to run renovate with an environment variable that says "please don't run", but same option is available from `RENOVATE_CONFIG` and `RENOVATE_CONFIG_FILE` (e.g. `config.js`). 
These option are simply wrappers to the feature request of simply exposing `RENOVATE_ENABLED=false`

Needed for the hosted app (simplicity)

**Comment by secustor** [comment]
Out of curiosity, can you share the use case?

**Reply by nabeelsaabna** [reply]
The hosted app backend includes a database (for many things) and with the new frontend portal the user will have the ability to disable Renovate for specific repository while keeping it installed in the GitHub App.
In general no jobs should be triggered for such repositories, but we need the `RENOVATE_ENABLED=false` as safety mechanism.

**Reply by Gabriel-Ladzaretti** [reply]
Resolved in:
 - https://github.com/renovatebot/renovate/pull/23516
